assets/schema_input.json

```
@@ -10,17 +10,21 @@
             "sample": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "Sample name must be provided and cannot contain spaces"
+                "errorMessage": "Sample name must be provided and cannot contain spaces",
+                "meta": ["sample"]
             },
             "datatype": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "Data type, and must be one of: 'pacbio'"
+                "errorMessage": "Data type, and must be one of: 'pacbio'",
```

```suggestion
                "errorMessage": "Data type, and must be one of: 'pacbio'",
                "enum": ["pacbio"],
```

This will force failure if datatype is not in the approved list of strings (currently pacbio only?)

---

Looks great!